initdb.d/04_grants.sql

```diff
@@ -12,18 +12,28 @@
 CREATE USER lega_in;
 CREATE USER lega_out;
 
+-- Temporary rights for the sda schema
+GRANT USAGE ON SCHEMA sda TO lega_in, lega_out;
+GRANT ALL PRIVILEGES ON ALL TABLES    IN SCHEMA sda TO lega_in;
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA sda TO lega_in;
+GRANT ALL PRIVILEGES ON ALL TABLES    IN SCHEMA sda TO lega_out;
```

Comment:
we might want to restrict this to only views, i am pretty sure `lega_out` requires read only and it will not be inserting anything in the database

Reply:
yes! The final PR will have sound privileges :)